services/user.py

```diff
@@ -0,0 +1,52 @@
+from django.contrib.auth import get_user_model
+
+
+User = get_user_model()
```

Comment:
According to the checklist, you should use `get_user_model()` instead of `User` directly. You've imported the function but didn't use it in the rest of your code.